doc/nrf/protocols/lte/lte_ncs.rst

```diff
@@ -0,0 +1,64 @@
+.. _ug_cellular_ncs:
+
+Getting started with cellular products
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+For getting started with the cellular products, you can refer to the :ref:`ug_nrf91` documentation, which guides through the features, getting started steps, and advanced usage of the boards.
+For a self-paced, hands-on course, enroll in DevAcademy's `Cellular IoT Fundamentals course`_, which describes the LTE and NB-IoT technologies and the software architecture of a cellular IoT application based on Nordic Semiconductor nRF91 Series devices.
+
+The :ref:`nrf9160_ug_network_mode` section provides information about switching between LTE-M and NB-IoT.
+
+Cellular IoT in |NCS|
+*********************
+
+The following sections explain the different applications, libraries, and samples that use the cellular IoT products.
+
+Modem library
+=============
+
+.. include:: ../../app_dev/device_guides/nrf91/nrf91_features.rst
+    :start-after: nrf91_modem_lib_start
+    :end-before: nrf91_modem_lib_end
+
+Library support
+===============
+
+The following |NCS| libraries are used for cellular IoT:
+
+* :ref:`lib_modem`
+* :ref:`lib_networking`
+* :ref:`liblwm2m_carrier_readme` library
+* :ref:`mod_memfault` library
+
+Applications and samples
```

Comment:
Should the Asset Tracker Template be also mentioned here?

Reply:
Since the Asset Tracker Template is not part of NCS, it is not mentioned in this page. We have mentioned the template on the landing page of the [LTE](https://github.com/nrfconnect/sdk-nrf/pull/21280/commits/9cf97ccff137f989c6aa9782cae1050b865ad29f#diff-46a37a4a0827cfe863604be333fb38b5d0d6e3c64fe040416fa49b96ae1456f7) and [Developing with guides](https://github.com/nrfconnect/sdk-nrf/pull/21280/commits/9cf97ccff137f989c6aa9782cae1050b865ad29f#diff-59c08dcb208d24fb0cab2c2f0a51facdcc160f7ce138566270d0280b3514f914).